core/__snapshot__/com.karumi.kotlinsnapshot.core.KotlinSerializationTest_should serialize a String.snap

```diff
@@ -1 +1 @@
-Simple string
\ No newline at end of file
+"Simple string"
```

Comment:
Same here, if I have this string:  

```
{
  "class": "Request",
  "isSuccess": false,
  "networkError": {
    "class": "NotFound"
  }
}
```

I don't want to have `"{\n\"class\": \"Request\", ... "`

Reply:
Fixed now, you can read my last comment about serializing raw not nested strings.